### ISSUE TITLE
How to evaluate the accuracy of validation datasets?

### ISSUE THREAD

**Issue by chriszxk** [Deci-AI/super-gradients#1795]
### 💡 Your Question

Yes, I  have browsed the previous question, no answer.

### Versions

Hi,
The accuracy of the validation dataset will be computed when we train YOLO_NAS, and get the best_weight.pth. 

So how do we evaluate the accuracy of testing datasets using the best_weight.pth. Can you provide a script for us or any guidance?

Thank you

**Comment by ani-mal** [comment]
You can use [evaluate_from_recipe.py](https://github.com/Deci-AI/super-gradients/blob/d5a85fd318f4137806c37d73120f905e0d51f6a7/src/super_gradients/evaluate_from_recipe.py#L18)

Same concept as the other recipes, where on the recipe `.yaml` file you specify the dataloaders, set the dataset parameters, etc.

if you don't want to use the recipe, you can look on how the `evaluate_from_recipe` implementation looks like:

**Comment by BloodAxe** [comment]
Alternative option to what @ani-mal suggested is to use `trainer.test(...)` method.
See https://github.com/Deci-AI/super-gradients/blob/56de963ef40cd0e4e1c437b90536735ce7f71ba3/documentation/source/Data.md?plain=1#L63
